Makefile

```diff
@@ -14,14 +16,55 @@ access-mattermost:
 access-pagerduty:
 	make -C access/pagerduty
 
-.PHONY:
+.PHONY: access-example
 access-example:
 	go build -o build/access-example ./access/example
 
-.PHONY:
+# Run all tests
+.PHONY: test
+test:
+	go test -count 1 ./...
+
+# Individual releases
+.PHONY: release/access-slackbot
 release/access-slackbot:
-	make -C access/slackbot release
+	make -C access/slackbot clean release
 
 .PHONY: release/access-jirabot
 release/access-jirabot:
-	make -C access/jirabot release
+	make -C access/jirabot clean release
+
+.PHONY: release/access-mattermost
+release/access-mattermost:
+	make -C access/mattermost clean release
+
+.PHONY: release/access-pagerduty
+release/access-pagerduty:
+	make -C access/pagerduty clean release
+
+# Run all releases
+.PHONY: releases
+releases: release/access-slackbot release/access-jirabot release/access-mattermost release/access-pagerduty
+
+.PHONY: get-deps
+get-deps:
+	go get -v -t -d ./...
```

Comment:
isn't everything in this repo vendored?

Reply:
Yep. My understanding of what vendoring does is still a little broken, I think. Essentially I should only need to use `go get` if there are things which aren't supplied by `vendor/`?

Reply:
Yes. In practice you should only be in one of two states:
- nothing is vendored and you use `go get` (possibly with go modules)
- everything is vendored
  - if using go modules, you use `go get` to add new dependencies or change versions of existing dependencies; this is only done by humans and checked in via PRs, automation never calls `go get`

Vendoring is preferred.
Since this repo already does vendoring, all dependencies should be there.

Reply:
Got it. Removed the `go get` commands.